src/schema/cfm/api.yaml

```diff
@@ -0,0 +1,437 @@
+
+        tags:
+          description: Arbitrary words to associate with this fridge.
+          type: array
+          items:
+            type: string
+          example: [harlem, halal, kashrut]
+
+        location:
+          $ref: '#/components/schemas/Location'
+
+        maintainer:
+          $ref: '#/components/schemas/Maintainer'
+
+        notes:
+          description: Notes about the fridge.
+          type: string
+          example: Next to Lot Radio.
+
+        photoURL:
+          description: Fully qualified URL path to a picture of the fridge.
+          type: string
+          format: uri
+          nullable: true
+          example: https://s3.amazonaws.com/bucket/path/fridge.webp
+
+        verified:
+          description: Signifies that the fridge data has been verified by the fridge maintainer.
+          type: boolean
+          default: false
+          example: false
+
+    Location:
+      type: object
+
+      required:
+        - street
+        - zip
+        - geoLat
+        - geoLng
+
+      properties:
+        street:
+          type: string
+          example: 17 Nassau Ave
+
+        city:
+          type: string
+          example: Brooklyn
+
+        state:
+          type: string
+          example: NY
+
+        zip:
+          type: string
+          example: 11222
+
+        geoLat:
+          description: Geographic latitude.
+          type: number
+          readOnly: true
+          example: 40.722814
+
+        geoLng:
+          description: Geographic longitude.
+          type: number
+          readOnly: true
+          example: -73.95405
+
+    Maintainer:
+      type: object
+
+      properties:
+        name:
+          description: Name of the contact person.
+          type: string
+          nullable: true
+          example: Emily Ward
+
+        organization:
+          description: Name of the organization.
+          type: string
+          nullable: true
+          example: Food For People
+
+        phone:
+          type: string
+          format: phone
+          nullable: true
+          example: 782-654-4748
+
+        email:
+          type: string
+          format: email
+          nullable: true
+          example: eward@example.net
+
+        website:
+          type: string
+          format: uri
+          nullable: true
+          example: http://gray.com/
+
+        instagram:
+          type: string
+          nullable: true
+          example: foodforpeople
+
+    Report:
+      type: object
+
+      required:
+        - timestamp
+        - operation
+
+      properties:
+        timestamp:
+          description: The ISO formatted date/time of this report.
+          type: string
+          format: date-time
+          readOnly: true
+          example: '2022-03-29T18:10:38.547Z'
+
+        condition:
+          description: The condition of the fridge. Fridge is ...
+          type: string
+          enum: [dirty, out of order, not at location]
```

Comment:
Just a thought for this use case

I think there should be a status on the backend of "good" but the frontend doesn't have to send this to us? We can just assume that the status is "good" if a condition is not passed in?